index.html

```diff
@@ -182,32 +191,34 @@ <h2> Let's Connect</h2>
                     <button type="button" id='project-1' class="see-more-button">
                         Get my resume
                     </button>
-                    
-                    <div class="skills-set">
-                    <div class="languages-title">
-                        <p>languages</p>
-                        <img src="images/down-arrow.svg" alt="down arrow icon">
-                    </div>
-                    <ul class="languages-vertical">
-                        <li class="languages">
-                            <img src="images/js.svg" alt="java script icon">
-                            <p>javascript</p></li>
-                        <li class="languages">
-                            <img src="images/html5.svg" alt="css icon">
-                            <p>html</p></li>
-                        <li class="languages">
-                            <img src="images/css3.svg" alt="java script icon">
-                            <p>css</p></li>
-                    </ul>
-                    
-                    <div class="not-expanded">
-                        <p>Frameworks</p>
-                        <img src="images/icons/horizontal-arrow.svg" alt="horizontal arrow">
-                    </div>
-                    <div class="not-expanded">
-                        <p>skills</p>
-                        <img src="images/icons/horizontal-arrow.svg" alt="horizontal arrow">
-                    </div>
+            </div>
+
+            <div class="skills-set">
+                <div class="languages-title">
+                    <p>languages</p>
+                    <img src="images/down-arrow.svg" alt="down arrow icon">
+                </div>
+
+                <ul class="languages-vertical">
+                    <li class="languages">
+                        <img src="images/js.svg" alt="java script icon">
+                        <p>javascript</p></li>
+                    <li class="languages">
+                        <img src="images/html5.svg" alt="css icon">
+                        <p>html</p></li>
+                    <li class="languages">
+                        <img src="images/css3.svg" alt="java script icon">
+                        <p>css</p></li>
+                </ul>
+            
+                <div class="not-expanded">
+                    <p>Frameworks</p>
+                    <img src="images/icons/horizontal-arrow.svg" alt="horizontal arrow">
+                </div>
+                <div class="not-expanded">
+                    <p>skills</p>
+                    <img src="images/icons/horizontal-arrow.svg" alt="horizontal arrow">
```

Comment:
- [ ]  You should position your element in the right position as they in Figma design. Check the space you have in your elements, so that it resemble to the one from Figma design.
![image](https://user-images.githubusercontent.com/43877689/177987557-faf266a3-19ec-46a1-bd68-c585320c00a4.png)